scaling_performance/host_practices.adoc

```diff
@@ -169,6 +169,8 @@ endif::[]
 etcd processes are typically memory intensive. Master / API server processes are CPU intensive. This makes them a reasonable co-location pair within a single machine or virtual machine (VM). Optimize communication between etcd and master hosts either by co-locating them on the same host, or providing a dedicated network.
 ====
 
+For a large and dense cluster, etcd may suffer from poor performance if the keyspace grows excessively large and exceeds the space quota. Periodic maintaince of etcd including defragmentaion needs to be done to free up the space for the backend. It is highly recommended to monitor prometheus for etcd metrics and defrag it when needed before etcd raises a cluster-wide alarm that puts the cluster into a maintenance mode which only accepts key reads and deletes. Some of the key metrics to monitor are `etcd_server_quota_backend_bytes` which is the current quota limit, `etcd_mvcc_db_total_size_in_use_in_bytes` indicates the actual database usage after a history compaction while `etcd_debugging_mvcc_db_total_size_in_bytes` shows the database size including free space waiting for defragmentation.
```

Comment:
Suggestions:  
- change "free up space in the backend" to "free up space in the data store"
- capitalize Prometheus
- `etcd_mvcc_db_total_size_in_use_in_bytes` which indicates
- change "while `etcd_debugging_mvcc_db_total_size_in_bytes` shows" to "and `etcd_debugging_mvcc_db_total_size_in_bytes` which shows"

Reply:
Made the changes requested. PTAL, thanks.